packages/react/src/lib/getOrGenerateIdFromShorthand.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { ShorthandValue } from '../types'
+
+const getOrGenerateIdFromShorthand = (
+  idPrefix: string,
```

Comment:
```suggestion
  prefix: string,
```

nit: feel free to take own decision 😺